src/Driver/SolverTypes/MISSolverType.jl

```diff
@@ -34,14 +34,23 @@ fast and slow dynamics respectively, depending on the state `Q`.
     Default: `MIS2`
 - `fast_method` (Function): Function defining the fast solver.
     Default: `LSRK54CarpenterKennedy`
-- `nsubsteps` (Int): Integer denoting the total number of times
+- `nsubsteps` (Tuple): Tuple denoting the total number of times
```

Comment:
We should document how to specify sub stepping in nested solvers. This is also a problem when using the multirate solver type, so this might be an opportunity to finally pin down something concrete!